### DISCUSSION TITLE
Layer opening animation

### DISCUSSION THREAD

**Discussion by alexandremjacques** [unpoly/unpoly#447] [Q&A] [answered]
Is it possible to animate new layer openings? I was willing to have a quick fade-in on layers.

**Comment by triskweline** [comment, marked as answer]
You can use the `[up-animate]` attribute or `{ animation }` option to animate the layer opening. You can choose a [predefined animation](https://unpoly.com/up.animate#named-animations) or [define your own](https://unpoly.com/up.animate).

You can also configure a default animation in `up.layer.config.overlay.openAnimation`.

Also layers already fade in and out by default, so you may have disabled animations in your JavaScript (`up.motion.config.enabled`) or in your OS settings.

**Reply by alexandremjacques** [reply]
That was it. My OS settings were confirgured with reduced motion.

Tks!

**Reply by triskweline** [reply]
You can force-enable animations with `up.motion.config.enabled = true`.
